index.html

```diff
@@ -11,7 +11,8 @@ <h1 class="display-3">GDI SF's Summer 2016 Students</h1>
         <p class="lead">This is a directory of portfolios websites created by GDI's Bootstrap &amp; Hosting Workshop students. </p>
       </div>
 
- <div class="card">
+ <div class="card-columns">
+    <div class="card">
```

Comment:
It looks like you added a second `<div class="card-columns">`, make sure your card is a child of the existing card-columns div and we're good to go